### ISSUE TITLE
✨ [Feature] Build admin platform

### ISSUE THREAD

**Issue by hwbrzzl** [goravel/goravel#545]
### Before feedback (在反馈之前)

- [X] There are no features that I want to submit in Issues (当前 Issues 中没有我将要提交的新功能)

### Describe Feature (描述功能)

Many users need a basic admin platform, including Authentication and Authorization. We can offer a VUE + Goravel architecture, then React + Goravel. 

The architecture should: 

1. Support adding modules conveniently.
2. Add E2E and integration test.
3. Convenient for secondary development.
4. Well-documented.

**Comment by hulutech-web** [comment]
可以先做一个，文章系统，或者官网类的

**Comment by hwbrzzl** [comment]
Yes, we are building a market: https://github.com/goravel-ecosystem/market-backend
